src/msg/async/Event.h

```diff
@@ -153,7 +153,7 @@ class EventCenter {
 
  private:
   CephContext *cct;
-  std::string type;
+  std::string_view type;
```

Comment:
This deserves a comment on how we prevent dangling string references. I'm not even sure it's not possible...

Reply:
yeah, i should have done this -- this `type` originally is passed down as a string literal. but after a second thought, i think it's not worthwhile to trade the couple bytes with the readability. i will drop this commit.